probe_scraper/glean_checks.py

```diff
@@ -202,15 +211,34 @@ def check_for_expired_metrics(
         for metric_name, metric in metrics.items():
             if metric["expires"] == "never":
                 continue
+
+            # `expires` field supports manual expiry, too.
+            if metric["expires"] == "expired":
+                expired_metrics.append(f"- {metric_name} manually expired")
+                addresses.update(metric["notification_emails"])
```

Comment:
This duplicates code that comes later. I wonder if it would be better to define a "has expired" function and then use it here? Might make this easier to follow

Reply:
It'd have to return the custom messages, but it could work. I'll see what I can do.